main.tex

```diff
@@ -777,13 +818,9 @@
 
 \end{enumerate}
 
-\item  An Owner or Occupier of a Lot without the consent of the Owners Corporation must not move any bulky goods or heavy goods, into or out of the Development except between the hours of 9.00AM and 5.00PM on Business Days and then only when the lift protection covers have been installed.
-
-\item  An Owner or Occupier must not hold the lift doors open and/or prevent the lift doors from closing for any lengthy periods of time which interferes with the normal operation of the lifts.
+\item  An Owner or Occupier of a Lot without the consent of the Owners Corporation must not move any bulky goods or heavy goods, into or out of the Development except between the hours of 7.30AM and 6.00PM on weekdays and 10AM and 5PM on weekends, and then only when the lift protection covers have been installed.
 
-\item  If an Owner or Occupier wishes to move into the Development on a weekend or after hours, that Owner or Occupier must pay the costs, expenses and charges that the Owners Corporation Manager has determined to employ or contract staff to oversee the move to ensure the Common Property is protected.
```

Comment:
When moving in, there was a completely useless guy employed by Brunswick Yard, directing me around, without knowing the full story about what was going on. He didn't know his way around and was mixed up about the schedule.

There was absolutely no point to him being there, and I don't feel we should be requiring such people to be hired again.